helpers/build_packages.sh

```diff
@@ -327,13 +327,10 @@ fi
 if [ "$BUILDGG" = "1" ]; then
 
     # look for either DEVICE or HABUILD_DEVICE files, do not use wildcards as there could be other variants
-    pattern_lookup=$(ls "$ANDROID_ROOT"/hybris/droid-configs/patterns/jolla-hw-adaptation-{$DEVICE,$HABUILD_DEVICE}.yaml 2>/dev/null | uniq)
-    metapackage_lookup=$(ls "$ANDROID_ROOT"/hybris/droid-configs/patterns/patterns-sailfish-device-adaptation-{$DEVICE,$HABUILD_DEVICE}.inc 2>/dev/null | uniq)
+    pattern_lookup=$(ls "$ANDROID_ROOT"/hybris/droid-configs/patterns/patterns-sailfish-device-adaptation-{$DEVICE,$HABUILD_DEVICE}.inc 2>/dev/null | uniq)
```

Comment:
I wonder if this will be confusing as those are not patterns anymore?

Reply:
We name meta-packages as patterns-*, thus let's stick to conventional naming (abstraction FTW).